src/pointer-analysis/value_set_analysis.h

```diff
@@ -14,32 +14,90 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #define USE_DEPRECATED_STATIC_ANALYSIS_H
 #include <analyses/static_analysis.h>
+#include <util/xml_expr.h>
+#include <util/xml.h>
 
 #include "value_set_domain.h"
 #include "value_sets.h"
+#include "value_set.h"
 
 class xmlt;
 
-class value_set_analysist:
+template<class Value_Sett>
```

Comment:
Plz no camel snake case. `ValueSet`?

Reply:
I don't think adopting a different naming convention is a good way of disambiguating it from the existing `value_sett`. Perhaps `selected_value_sett`? Perhaps there is a better term. `value_set_domaint`?